broker/broker-route-manager/src/main/java/org/codice/ddf/broker/route/manager/DynamicRouteDeployer.java

```diff
@@ -87,4 +95,12 @@ public void uninstall(File file) throws Exception {
   public boolean canHandle(File file) {
     return file.getName().endsWith(".xml");
   }
+
+  private void addRouteDefinitions(List<RouteDefinition> routeDefinitions) {
+    try {
+      camelContext.addRouteDefinitions(routeDefinitions);
+    } catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
❗️ We should not be throwing RuntimeException

Reply:
What exception do you suggest we replace it with?A custom exception or is there a better alternative?